### ISSUE TITLE
Level of blinds in pop-up window

### ISSUE THREAD

**Issue by braindead1** [Zefau/ioBroker.jarvis#14]
For blinds the level value is shown next to the buttons (up and down) in the pop-up window. This is not only confusing due to little space, but also inconsistent, as the level value is not show for dimmable lights.

Proposal: Hide the label value and add some space between the slider and the buttons to make it look similar the the pop-up of dimmable lights.

![blind](https://user-images.githubusercontent.com/2136249/76677490-33357b00-65cf-11ea-8faf-26adcb1ae600.png)

**Comment by Zefau** [comment]
Thanks for all the bug reporting and support! And thanks for the donation as well! Highly appreciated!

**Comment by Zefau** [comment]
Should be fixed with v0.0.6. Could you please test and verify? Thanks

**Comment by braindead1** [comment]
I installed v0.0.6. Although it looks nicer now with the space between the slider and the value, I noticed the following:

- The slider is not slideable anymore. To change the value, you have to click on the slider at the value you want to set. If you want to set the slider to a specific value (e.g. 42%), you will most probable have to try it more than once.
- The sliders in pop-up windows of dimmable lights and shutters look different. Unless there is a specific reason, you should make them look the same to be consistent within the interface.

**Comment by Zefau** [comment]
In what regard to they look different?

Light:
![image](https://user-images.githubusercontent.com/14368811/76699149-af50c100-66aa-11ea-9d31-32f13724c25b.png)

Shutters:
![image](https://user-images.githubusercontent.com/14368811/76699153-baa3ec80-66aa-11ea-9160-2c2d9d884def.png)


The shutters additionally have the actions for up an down. The interface though is the same, isn't it?

**Comment by braindead1** [comment]
Oh, I guess that my dimmers are not configured correctly, as mine look like this:

![lights](https://user-images.githubusercontent.com/2136249/76699200-706f3b00-66ab-11ea-99d2-7c59b813afde.png)

And this is the JSON:

`{"badezimmer_deckenspots": {
    "name": "Deckenspots",
    "room": "badezimmer",
    "function": "light",
    "states": {
      "level": {
        "state": "hm-rpc.2.XXXXXXXXXXXXXX.4.LEVEL",
        "action": "hm-rpc.2.XXXXXXXXXXXXXX.4.LEVEL"
      }
    },
    "jarvis": {
      "group": "licht_erdgeschoss",
      "subgroup": "Badezimmer"
    }
  }
}`

**Comment by Zefau** [comment]
Looks ok to me? What's wrong with it?

jarvis has added the power on/off button to _Helligkeit_ (`level`) because the `power` state is not given.

If you add `power` it will add _Licht_ as seen in my screenshot.

See the first Wiki page (https://github.com/Zefau/ioBroker.jarvis/wiki ) for my full configuration example.
A full light configuration could be:
```
   "light_bedroom_ceiling":{
      "name":"Schlafzimmer",
      "room":"bedroom",
      "function":"light",
      "states":{
         "power":{
            "state":{
               "node":"hue-extended.0.lights.009-schlafzimmer_deckenlicht.action.on",
               "subscribe":true
            },
            "action":"hue-extended.0.lights.009-schlafzimmer_deckenlicht.action.on"
         },
         "level":{
            "state":"hue-extended.0.lights.009-schlafzimmer_deckenlicht.action.level",
            "action":{
               "node":"hue-extended.0.lights.009-schlafzimmer_deckenlicht.action.level",
               "cloud":true
            }
         },
         "colorTemperature":{
            "state":"hue-extended.0.lights.009-schlafzimmer_deckenlicht.action.colorTemperature",
            "action":{
               "node":"hue-extended.0.lights.009-schlafzimmer_deckenlicht.action.colorTemperature",
               "cloud":true
            }
         }
      },
      "jarvis":{
         "group":"Licht",
         "subgroup":"Schlafzimmer",
         "sort":31
      }
   },
```

**Comment by braindead1** [comment]
Thank you for the explanation. I added "power" as additional state and as I'm using HomeMatic IP dimmers, there is no separate state for on/off. I used the level state for "power" and now it's working as described in the wiki.

The problem that the sliders are not slideable remains.

**Comment by Zefau** [comment]
Should be fixed with v0.0.7. Could you please test and verify? Thanks

**Comment by braindead1** [comment]
In v0.0.7 the problem is fixed.
